### ISSUE TITLE
Sent Time is in UTC in reply() quoted text

### ISSUE THREAD

**Issue by austdav89** [microsoftgraph/msgraph-sdk-dotnet#1108]
**Describe the bug**
When using the Reply() method everything works as it should, but a customer notified us that the Sent Time in the reply chain is not correct and is in UTC. I'm not sure if this is a bug or not, but I'm not sure how to correct this.

Reply receipt: Sent should read 4:11:52 PM (Original emails received date)

```
From: ######## - #### #### ##### <#####@####.com>
Sent: Tuesday, August 17, 2021 9:11:52 PM
To: #########
Subject: #######
```

**To Reproduce**

//API POST to send email
await _graphClient.Users[emailAccount].Messages[messageId]
    .Reply(message, body)
    .Request()
    .PostAsync();

**Expected behavior**
Have the Sent: header in the reply shown as it is when it originally came in. Currently it's being convert to UTC with no offset.

 
[AB#10776](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/10776)

**Comment by andrueastman** [comment]
Hey @austdav89, 

Thanks for opening this issue.
I have also tried the same on Graph Explorer and I also get similar results. 

I have forwarded the question at the link below so that the relevant API owners may give us feedback on what needs be done to resolve this or whether it a service bug or not. 

https://docs.microsoft.com/en-us/answers/questions/520090/sent-time-is-in-utc-when-using-createreply.html

In the meantime, I will close this issue so that we may follow it up from there and re-open it if there is a fix that may need to be done on the client library.
